app/views/admin/enterprises/form/_contact.html.haml

```diff
@@ -6,8 +6,13 @@
 .row
   .alpha.three.columns
     = f.label :email_address, t('.email_address')
-    %div{'ofn-with-tip' => t('.email_address_tip')}
-      %a= t('admin.whats_this')
+    %div{ data: { controller: "tooltip", "tooltip-placement-value": "top-start" } }
+      %a{"data-tooltip-target": "element", "data-action": "mouseenter->tooltip#showTooltip mouseleave->tooltip#hideTooltip"}= t('admin.whats_this')
+      .tooltip-container
+        .tooltip{"data-tooltip-target": "tooltip"}
+          = t('.email_address_tip')
+          .arrow{"data-tooltip-target": "arrow"}
+
```

Comment:
It's good to start with explicit code like this and copying it to another place is okay, too. But my rule of thumb is to de-duplicate code before creating a third or fourth copy. This could go into a partial which takes the text as variable.